### DISCUSSION TITLE
Can i use this to dim the lights at certain times, not based on the sun?

### DISCUSSION THREAD

**Discussion by Jleagle** [basnijholt/adaptive-lighting#619] [Q&A] [answered]
I want my lights to slowly dim from 20:00 to 22:00 each night. Is this possible?

I have tried to override the sunset times but it's never quite right. Is this what sleep mode is for?

Thanks!

**Comment by backcountrymountains** [comment]
This is something that you might be able to do with the my fork by setting sunset to 20:00 and sleep_time to 22:00. See #632

**Comment by Jleagle** [comment, marked as answer]
This is how i ended up doing it.

```adaptive_lighting:
  - name: "bathroom"
    brightness_mode: "linear"
    brightness_mode_time_light: "3600"
    brightness_mode_time_dark: "0"
    detect_non_ha_changes: true
    min_brightness: 1
    sunrise_time: "06:00:00"
    sunset_time: "20:00:00"
    lights:
      - "light.shellydimmer2_48551900b4e6"
